helloworld/test/index-test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const chai = require('chai');
+const greet = require('../index.js');
```

Comment:
I switched them now for re-enforcement and pushed them up. I appreciate the thorough commenting. Thank You.